features/community/hooks/useReportReview.ts

```
@@ -0,0 +1,47 @@
+        switch (error.statusCode) {
+          case 400:
+            // 유효성 검사 에러가 있을 경우 상세 메시지 표시
+            const validationMsg = error.validation
+              ? Object.values(error.validation).join(', ')
+              : error.message;
+            alert(`신고 내용을 확인해주세요: ${validationMsg}`);
+            break;
+
+          case 401:
+            alert('로그인이 필요한 기능입니다.');
+            // 필요 시 로그인 모달을 띄우거나 페이지 이동
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)